### DISCUSSION TITLE
Option to delete Post Type record when relationship is removed

### DISCUSSION THREAD

**Discussion by khill07** [pods-framework/pods#6704] [Ideas]
## Issue Overview
If you have 2 custom post types, one acts as the parent the other is linked through a one to many child relationship, when removing the child record from the parent, ONLY the relationship is removed.  The child record still exists.  It would be helpful if there was an option to delete the child record alone with the relationship.

## Expected Behavior
When a relationship is setup between with another post types as a Multiple Select, List View, add 2 check boxes:
1. Clear Relationship (this acts as the same functionality that exists today)
2. Trash Record (this would move child record to the trash and remove the relationship)

## Current Behavior
Only the relationship between parent and child is removed

## Workaround or Alternate Solution Until Bug is Addressed
Create a hook on the parent post type and trash any orphaned child records

**Comment by quasel** [comment]
Workaround: Change type to flexible relationship - then you can just thrash the item on the edit modal
